src/libs/pr.ts

```diff
@@ -20,7 +34,7 @@ export async function handlePullRequestMessage(
 
   const summary = '<summary>Pulumi report</summary>';
 
-  const rawBody = output.substring(0, 64_000);
+  const rawBody = ansiToHtml(output, { maxLength: 64_000 });
```

Comment:
Since this PR touches this code, would you mind pulling this value into a constant so it's not a ["magic number"](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544)? I'd appreciate it :D